src/gui/inputdevices/PenInputHandler.cpp

```diff
@@ -302,7 +286,7 @@ auto PenInputHandler::actionEnd(InputEvent* event) -> bool {
             if (!this->lastHitEvent) {
                 return false;
             }
-            currentPage = getPageAtCurrentPosition(this->lastHitEvent);
+            currentPage = getPageAtCurrentPosition(&this->lastHitEvent);
```

Comment:
`AbstractInputHandler::getPageAtCurrentPosition` has a null check for the event argument. This could potentially change the semantics since `lastHitEvent` is now just initialized to some garbage event data.

Reply:
For that exact case, I added an `operator bool()` which returns `true`, if the underlying `GtkEvent*` is valid.

Reply:
fixed that thank you for pointing that out